examples/transport_layer/CMakeLists.txt

```diff
@@ -1,7 +1,10 @@
 cmake_minimum_required(VERSION 3.16)
+project(transport_layer_example)
 
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+find_package(isobus CONFIG)
+find_package(Threads REQUIRED)
 
 add_executable(TransportLayerExampleTarget main.cpp)
-target_link_libraries(TransportLayerExampleTarget ${PROJECT_NAME}::Isobus ${PROJECT_NAME}::HardwareIntegration Threads::Threads ${PROJECT_NAME}::SystemTiming)
```

Comment:
We can no longer infer the project name for the examples is the same as the targets we want to link to, so spell it out.